### PR TITLE
Improve startup script

### PR DESCRIPTION
**Pull request by smahmed776**
Used node package `concurrently` to start all backend services in a single terminal tab.

### CODE REVIEW

**Comment by itsjoeoui** [comment]
Some changes:
1. Please make sure Poetry is installed, it will be used to manage Python environments: https://python-poetry.org/
2. Run `npm run init` to initialize all dependencies (only need to be done once)
3. Run `npm run flojoy` to start the app.

**Review by JeffDotPng — APPROVED** [review]

File: flojoy [hunk 1]
```diff
@@ -329,28 +329,28 @@ echo ''
 # Initializing FLOJOY-WATCH RQ Worker
 cd $CWD
 
-npx ttab -t 'Flojoy-watch RQ Worker' "$venvCmd; export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES; rq worker flojoy-watch"
+# npx ttab -t 'Flojoy-watch RQ Worker' "$venvCmd; export OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES; rq worker flojoy-watch"
```
Comment (topasif) [inline flojoy:332]: we can get rid of all these commented out lines

**Review by topasif — APPROVED** [review]

File: flojoy (hunk 1 above)
Comment (itsjoeoui) [reply flojoy:332]: I think we can get rid of the old script entirely, right?

Comment (dev-nested) [reply flojoy:332]: yes I think so

**Comment by jackparmer** [comment]
@itsjoeoui @smahmed776 
is this ready for review?

**Comment by itsjoeoui** [comment]
> @itsjoeoui @smahmed776 is this ready for review? (after the merge conflicts are fixed)

Yes, should be good to go now.

**Pull request merged**